client/src/pages/material-details.tsx

```diff
@@ -0,0 +1,67 @@
+// material-details.tsx
+import React, { useEffect, useState } from 'react';
+import { Card, CardContent, Grid, Typography, Divider, Button,useTheme, alpha  } from '@mui/material';
+import { useParams } from 'react-router-dom';
+import FileComponent from '../components/course-material-details/course-material-details';
+import { Material, materials } from '../_mocks/materials';
+import { useResponsive } from '../hooks/useResponsive';
+import { useAuth } from '../hooks/useAuth'
+const MaterialDetailsPage = () => {
+  const { id } = useParams();
+  const [materialDetails, setMaterialDetails] = useState<Material | null>(null);
+  const { authUser } = useAuth()
+
+  useEffect(() => {
+    // Obtener los detalles del material usando el ID
+    const getMaterialDetails = async () => {
+      // Buscar el material por ID
+      const foundMaterial = materials.find((material) => material.id === id);
+
+      if (foundMaterial) {
+        setMaterialDetails(foundMaterial);
+      }
+    };
+
+    getMaterialDetails();
+  }, [authUser, id]);
+
+    const isSmallScreen = useResponsive('down', 'sm')
+    const handleDownload = () => {
+      const link = document.createElement('a');
```

Comment:
Could you add the option `link.target = '_blank';` to open a new tab when clicking on download?